app/views/layouts/dashboard.html.erb

```diff
@@ -0,0 +1,43 @@
+<% content_for?(:page_title) ? yield(:page_title) : fallback_title %>
+
+<% content_for(:head) do %>
+  <%= csrf_meta_tags %>
+  <%= csp_meta_tag %>
+<% end %>
+
+<% content_for(:analytics) do %>
+  <%= render partial: 'layouts/analytics' %>
+<% end %>
+
+<% content_for(:cookie_banner) do %>
+  <% render partial: 'layouts/cookie_banner/main', locals: { cookies_preferences_set: @cookie_preferences_set } %>
+<% end %>
+
+<% content_for(:service_name) do %>
+  <%= link_to t('assess_a_crime_form.service_name'), root_path,
+              class: 'govuk-header__link govuk-header__service-name', id: 'header-service-name' %>
+<% end %>
+
+<% content_for(:header_navigation) do %>
+  <% render partial: 'layouts/header_navigation' %>
+<% end %>
+
+<% content_for(:phase_banner) do %>
+  <% render partial: 'layouts/phase_banner' %>
+<% end %>
+
+<% content_for(:primary_navigation) do %>
+  <% render partial: 'layouts/dashboard/primary_navigation' %>
+<% end %>
+
+<% content_for(:content) do %>
+  <%= yield %>
+<% end %>
+
+<% content_for(:footer_links) do %>
+  <% render partial: 'layouts/footer_links' %>
+<% end %>
+
+<%# render partial: 'layouts/developer_tools' if FeatureFlags.developer_tools.enabled? %>
```

Comment:
I don't think we have a partial called this - I think this is commented out in every layout, and we should probably ditch it.